src/libsyntax/parse/mod.rs

```diff
@@ -33,7 +33,6 @@ pub mod attr;
 
 pub mod common;
 pub mod classify;
-pub mod obsolete;
```

Comment:
Then I guess we would also want to keep the latest special cases (enum variants: ClosureType, ForSized, Sized, ProcType, ProcExpr). I will update the PR with this changes.

Thanks for the feedback!